docs/Employer/5-Employment.md

```diff
@@ -72,7 +72,21 @@ Parameter | Type | Required | Description
           "amount": 10000000,
           "currency": "usd",
           "effective_date": "2019-03-01",
-        }
+        },
+        "income_history": [
+          {
+            "unit": "yearly",
+            "amount": 9000000,
+            "currency": "usd",
+            "effective_date": "2018-05-15",
+          },
+          {
+            "unit": "yearly",
+            "amount": 8000000,
+            "currency": "usd",
+            "effective_date": "2017-06-30",
```

Comment:
lol..can you set this as the individual's `start_date`? Right now the start date is _after_ this

Reply:
Good call out